frontend/src/app/components/FeaturedExperiment/FeaturedButton.js

```diff
@@ -78,14 +77,17 @@ export default class FeaturedButton extends Component<FeaturedButtonProps, Featu
       this.sendMetric(ev, {eventLabel: "Popup Featured privacy"});
     };
 
-    return (<LocalizedHtml id={this.l10nId("legal-notice")}
-      $title={title}>
-      <p className="main-install__legal">
-            By proceeding, you agree to the <a href="/terms" onClick={(ev) => this.sendMetric(ev, {eventLabel: "Open general terms"})}></a>
-            and <a href="/privacy" onClick={(ev) => this.sendMetric(ev, {eventLabel: "Open general privacy"})}></a> policies of Test Pilot and the
-        <a href="#" onClick={launchLegalModal}></a>.
-      </p>
-    </LocalizedHtml>);
+    return (<Localized id={this.l10nId("legal-notice")}
+      $title={title}
+      terms-link={<a href="/terms" onClick={(ev) => this.sendMetric(ev, {eventLabel: "Open general terms"})}></a>}
+      privacy-link={<a href="/privacy" onClick={(ev) => this.sendMetric(ev, {eventLabel: "Open general privacy"})}></a>}
+      modal-link={<a href="#" onClick={launchLegalModal}></a>}>
+        <p className="main-install__legal">
+            By proceeding, you agree to the <terms-link>terms</terms-link>{" "}
```

Comment:
I'm not quite sure if using custom elements like that is a good practice in React. The contents of `<p>` will be replaced anyways by the translation, but before it happens, this fragment of the render tree will be constructed by React, I believe.

Perhaps it would be better to use a regular string?

```jsx
        <p className="main-install__legal">
            {`By proceeding, you agree to the <terms-link>terms</terms-link>
            and <privacy-link>privacy</privacy-link> policies of Test Pilot and
            <modal-link>this experiment's privacy policy</modal-link>.`}
        </p>
```

In the future, `fluent-react` might be able to extract en-US string from such elements. Right now, it would be just easier on the CPU.